monero-outreach-docs/translations/pt-br/storyboard/tool_for_renewal_pt_br.md

```diff
@@ -0,0 +1,77 @@
+# A Ferramenta para Renovação
+**“Participe de um movimento para mudanças positivas com uma ferramenta que reestabelece o poder econômico e social das pessoas: Monero.”**
+8 de Junho de 2020
+
+O ano é 2020, e nós estamos assistindo o desenrolar de uma crise multifacetada. Temos uma pandemia global, o ressurgimento de tensões econômicas e sociais, divisões geopolíticas extremas, e o abuso de poder de governos e bancos centrais ao redor do mundo. Temos, a história nos mostra, os ingredientes para uma mudança transformadora. Se você apoia ou está entre os que estão sendo oprimidos por razões políticas, religiosas e raciais, convidamos você para participar em um movimento por mudança positiva com uma ferramenta que restaura o poder econômico e social das pessoas: Monero.
+
+Disfunção e agitação civil são frequentemente precursores para profundas mudanças. Tumultos na Roma antiga eram pressagios de estágios finais da república. Na América, independência ocorria depois de protestos, danos materiais, e respostas violentas por parte das autoridades contras os manifestantes (tal como o massacre de Boston). Protestos e desobediência civil  acompanhados de fortes mudanças nos direitos civis começaram nos anos 1960.
+
+Sistemas financeiros frequentemente são inextricavelmente ligados a agitação civil e mudança social. Os fundamentos da Guerra Revolucionária Americana incluíram o desejo dos colonos de controlar a taxação e o destino da economia.
+
+Como os bancos centrais começaram a imprimir dinheiro depois do completo desmantelamento do padrão ouro em 1971, nasceu então um novo sistema fiduciário que se tornaria o padrão monetário global (Dólar Americano), determinando o valor da maioria das outras moedas. Pessoas agora estão vendo as consequências não-intencionais deste sistema financeiro, como fragilidade sócio econômica, desigualdade e disparidade. Elas querem algo melhor do que equilíbrio de poder. Embora não precisamos de uma guerra revolucionária para alcança-lo. Nós podemos simplesmente optar pacificamente por escolher um sistema financeiro tecnológico que está sendo construído por e para as pessoas.
+
+Em 2009 uma pessoa ou grupo de pessoas de pseudônimo Satoshi Nakamoto lançou o Bitcoin, precursor do Monero pouco depois do grande colapso financeiro. No bloco genesis, o primeiro bloco da blockchain do Bitcoin, Satoshi gravou uma notícia contemporânea para registrar a época da criação do Bitcoin. “The Times 03/Jan/2009 Chanceller a beira de um segundo resgate aos bancos.” Esta mensagem original ressoa até hoje, como bancos centrais procuram sustentar a sitiada economia global.
```

Comment:
Here is where I found the mistake. The English version read like this:

- "In 2009 a pseudonymous person or group named Satoshi Nakamoto launched Monero’s precursor Bitcoin on the _**heals**_ of the great financial collapse."

Correct:

- "In 2009 a pseudonymous person or group named Satoshi Nakamoto launched Monero’s precursor Bitcoin on the _**heels**_ of the great financial collapse."

Thanks